english/swami-vivekananda/swami-vivekananda.md

```diff
@@ -6,10 +6,12 @@ But, whatever you do, don't touch a hot stove.
 
 But, whatever you do, don't use the toaster in the bath tub ...
 
-But, whatever you do, don't talk about your favorite type of "Bath Bomb" or 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.